operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -514,6 +515,71 @@ function createModelDomain() {
   trace "Exiting createModelDomain"
 }
 
+
+# Expands into the root directory the MII domain configuration, stored in one or more config maps
+function restoreDomainConfig() {
+  restoreEncodedTar "domainzip.secure" || return 1
+
+  chmod +x ${DOMAIN_HOME}/bin/*.sh ${DOMAIN_HOME}/*.sh  || return 1
+}
+
+# Expands into the root directory the MII primordial domain, stored in one or more config maps
+function restorePrimordialDomain() {
+  restoreEncodedTar "primordial_domainzip.secure" || return 1
+}
+
+# Restores the specified directory, targz'ed and stored in one or more config maps after base 64 encoding
```

Comment:
It'd be helpful to have a high level explanation of the encoding approach in this comment (the 'range' file's purpose, and the directory structure when its not '0 0'...).

Reply:
With your suggested change, I no longer need the range file, so I will remove the Java code that creates it.